internal/xds/bootstrap/jwtcreds/bundle.go

```diff
@@ -0,0 +1,76 @@
+/*
+ *
+ * Copyright 2025 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package jwtcreds implements JWT Call Credentials for XDS, configured via xDS
```

Comment:
This package's implementation should be much simpler if we have a separate interface for call credentials just like we have today for transport credentials, and not having to deal with the bundle. (this follows from my other comments in those appropriate places).